licenses.yaml

```diff
@@ -1668,1285 +1604,1459 @@ libraries:
-version: 0.0.9
-copyright: to original author or authors
-license_file_path: licenses/bin/aether-connector-okhttp.EPL1
-libraries:
-  - io.tesla.aether: aether-connector-okhttp
+module: web-console
+license_name: MIT License
+version: 3.0.4
+license_file_path: licenses/bin/react-lifecycles-compat.MIT
 
 ---
 
-name: Tesla Aether
+name: "react-popper"
 license_category: binary
-module: java-core
-license_name: Eclipse Public License 1.0
-version: 0.0.5
-copyright: to original author or authors
-license_file_path: licenses/bin/tesla-aether.EPL1
-libraries:
-  - io.tesla.aether: tesla-aether
+module: web-console
+license_name: MIT License
+copyright: Travis Arnold
+version: 1.3.3
+license_file_path: licenses/bin/react-popper.MIT
 
 ---
 
-name: Eclipse Aether libraries
+name: "react-router-dom"
 license_category: binary
-module: java-core
-license_name: Eclipse Public License 1.0
-version: 0.9.0.M2
-copyright: Sonatype, Inc.
-license_file_path: licenses/bin/aether-core.EPL1
-libraries:
-  - org.eclipse.aether: aether-api
-  - org.eclipse.aether: aether-connector-file
-  - org.eclipse.aether: aether-impl
-  - org.eclipse.aether: aether-spi
-  - org.eclipse.aether: aether-util
+module: web-console
+license_name: MIT License
+version: 5.0.1
+license_file_path: licenses/bin/react-router-dom.MIT
 
 ---
 
-name: Rhino
+name: "react-router"
 license_category: binary
-module: java-core
-license_name: Mozilla Public License Version 2.0
-version: 1.7R5
-copyright: Mozilla and individual contributors.
-license_file_path: licenses/bin/rhino.MPL2
-libraries:
-  - org.mozilla: rhino
+module: web-console
+license_name: MIT License
+version: 5.0.1
+license_file_path: licenses/bin/react-router.MIT
 
 ---
 
-name: \"Java Concurrency In Practice\" Book Annotations
+name: "react-splitter-layout"
 license_category: binary
-module: hadoop-client
-license_name: Creative Commons Attribution 2.5
-version: 1.0
-copyright: Brian Goetz and Tim Peierls
-license_file_path: licenses/bin/creative-commons-2.5.LICENSE
-libraries:
-  - net.jcip: jcip-annotations
+module: web-console
+license_name: MIT License
+copyright: Yang Liu
+version: 4.0.0
+license_file_path: licenses/bin/react-splitter-layout.MIT
 
 ---
 
-name: Reactive Streams
+name: "react-table"
 license_category: binary
-module: java-core
-license_name: Creative Commons CC0
-version: 1.0.2
-license_file_path: licenses/bin/reactive-streams.CC0
-libraries:
-  - org.reactivestreams: reactive-streams
+module: web-console
+license_name: MIT License
+version: 6.10.0
+license_file_path: licenses/bin/react-table.MIT
 
 ---
 
-name: a smear function adapted from MurmurHash3
-license_category: source
-module: java-core
-license_name: Public Domain
-copyright: Austin Appleby who has placed MurmurHash3 in the public domain (https://github.com/aappleby/smhasher/blob/master/src/MurmurHash3.cpp)
-source_paths:
-  - processing/src/main/java/org/apache/druid/query/groupby/epinephelinae/Groupers.java
+name: "react-transition-group"
```

Comment:
Missing copyright: https://github.com/reactjs/react-transition-group/blob/master/LICENSE. I guess it could be `React Community
Forked from React (https://github.com/facebook/react) Copyright 2013-present, Facebook, Inc.`.